sdk/nodejs/src/client.ts

```diff
@@ -296,6 +303,35 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    private trackSDKConfigEvent(
+        url: string,
+        responseTimeMS: number,
+        res?: Response,
+        err?: ResponseError,
+        reqEtag?: string,
+        reqLastModified?: string,
+    ): void {
+        this.eventQueue.queueEvent(
+            DVCPopulatedUserFromDevCycleUser({ user_id: this.clientUUID }),
+            {
+                type: 'sdkConfig',
+                target: url,
+                value: responseTimeMS,
+                metaData: {
+                    clientUUID: this.clientUUID,
+                    reqEtag,
+                    reqLastModified,
+                    resEtag: res?.headers.get('etag') || undefined,
```

Comment:
what's the purpose of `|| undefined`?

Reply:
`.get()` returns `null` if it doesn't find anything, would prefer it doesn't send the param in the JSON using `undefined`.

Reply:
use `??` then